turbinia/turbiniactl.py

```diff
@@ -724,23 +700,29 @@ def main():
     request.evidence.append(evidence_)
 
     if args.recipe:
-      if args.jobs_denylist or args.jobs_allowlist or args.recipe_config or args.filter_patterns_file:
+      if args.jobs_denylist or args.jobs_allowlist:
         raise TurbiniaException(
             'Specifying a recipe is incompatible with defining'
-            ' jobs allow/deny lists, filter patterns or recipe_config'
-            'parameters separately.')
-      recipe_obj = TurbiniaRecipe(
-          os.path.join(config.RECIPE_FILE_DIR, args.recipe))
+            ' jobs allow/deny lists parameters separately.')
+
+      if config.RECIPE_FILE_DIR:
+        recipe_obj = TurbiniaRecipe(
+            os.path.join(config.RECIPE_FILE_DIR, args.recipe))
+      else:
+        recipe_obj = TurbiniaRecipe(
+            os.path.join('../data/recipes', args.recipe))
```

Comment:
I think we might want to keep the dynamic setting of the recipe config dir to a follow-up PR since it's non-trivial to implement.  I think this line will try to look for the files relative to the current working directory rather than the directory relative to this python file.  Also, depending on the deployment mechanism, the data directory might not even be at this relative path.   Maybe for now we just error out if the recipe dir isn't set in the config.